lib/builders/latexmk.js

```diff
@@ -50,7 +50,7 @@ export default class LatexmkBuilder extends Builder {
     }
   }
 
-  constructArgs (filePath, jobname, shouldRebuild) {
+  constructArgs (filePath, action, jobname) {
```

Comment:
What about changing this to
```js
constructArgs (filePath, action = 'build', jobname = null)
```

Then we can remove the `action` value from the specs. Even if you leave those, I think it makes sense to provide defaults here. What do you think?